src/client/cli/argparser.cpp

```diff
@@ -85,10 +85,8 @@ auto verbosity_level_in(const QStringList& arguments)
             return 1;
         if (arg == QStringLiteral("-vv"))
             return 2;
-        if (arg == QStringLiteral("-vvv"))
+        if (QRegExp{"-vvv+"}.exactMatch(arg))
             return 3;
-        if (QRegExp{"-vvvv+"}.exactMatch(arg))
-            return 4;
     }
     return 0;
```

Comment:
I think this needs to change to `return 1`.

Reply:
In fact, this return line is never reached, because the function is called only when verbosity is set. I prefer to leave it as zero, for considering the case on which the function is called when verbosity is not set. WDYT?

Reply:
Sorry, I had not realized that the other levels would need to change too. I think this function should provide the verbosity corresponding to a command line, in any case (even if it is not used in all cases). So I suppose something like the following?

```c++
auto verbosity_level_in(const QStringList& arguments)
{
    for (const QString& arg : arguments)
    {
        if (arg == QStringLiteral("-v") || arg == QStringLiteral("--verbose"))
            return 2;
        if (arg == QStringLiteral("-vv"))
            return 3;
        if (arg == QStringLiteral("-vvv+"))
            return 4;
    }
    return 1;
}
```

It would be inconsistent to return 0 in the default case then.

Reply:
But that `return 1` is never reached. To be clear, let's abstract from this function and state which the levels should be:
- no verbosity arguments should be level 1
- `-v` or `--verbose` should be level 2
- `-vv` should be level 3
- `-vvv` or more should be level 4
Is that what you had in mind?

Additionally, we can count in the future for an option `--quiet` to force verbosity level 0.

Reply:
Oh, right, sorry I didn't see that you are proposing that this function must be always called. +1 to that. So we are speaking the same language.

Reply:
Yes, that's what I think we need. 

I understand that the last return is never reached *today*. However, a function should provide a consistent unit of functionality, without assuming how it is called. According to the function name, that functionality is to provide the `verbosity_level_in` a cmd line (as represented by a list of arguments). 

If there are no `-v`s, the verbosity in the command line should now become 1. If we add `--quiet` in the future, this function would then need to take that into account.

Reply:
Oh, crossed msgs. Well, I think ideally it would make sense for this function to be the only unit determining the verbosity. I don't need that to be the case with this PR, but if it is a small change, I suppose that doesn't harm.

Reply:
Of course, it doesn't harm.